submodules/eks/cluster.tf

```diff
@@ -72,6 +72,17 @@ resource "aws_eks_cluster" "this" {
   }
 }
 
+data "tls_certificate" "cluster_tls_certificate" {
+  url = aws_eks_cluster.this.identity[0].oidc[0].issuer
```

Comment:
Can we do something like `try(aws_eks_cluster.this.identity[0].oidc[0].issuer, "")` here? In some recent operations I've seen index errors on things like this when the reference/attribute is null.

Reply:
Thats a property on eks I dont expect it to ever change and if it does I rather it blows up.

Reply:
I'm not sure I agree, but I'm not going to hold things up because of that.